vspace/pom.xml

```diff
@@ -358,11 +365,11 @@
             <artifactId>tika-core</artifactId>
             <version>1.22</version>
         </dependency>
-       <dependency>
-		    <groupId>javax.annotation</groupId>
-		    <artifactId>javax.annotation-api</artifactId>
-		    <version>1.3.2</version>
-   		</dependency>
```

Comment:
is there a reason this is being deleted?

Reply:
This lines are already present in line number 102 to 106. I removed this because of code clonflicts